### PR TITLE
PAINTROID-365 Fastlane Screengrab Setup

### PR DESCRIPTION
**Pull request by ThorstenBandel**
Creating a Setup for Fastlane Screengrabs to generate Screenshots for Google Play Store and Marketing
To run the test correctly use fastlanes screengrab in ruby shell in the root folder of Paintroid. Screenshots will be generated in the folder fastlane/metadata/..

Only screenshot tests will be automatically run and languages can be changed in the file "fastlane/Screengrabfile"

*Please enter a short description of your pull request and add a reference to the Jira ticket.*

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [ ] Include the name of the Jira ticket in the PR’s title
- [ ] Include a summary of the changes plus the relevant context
- [ ] Choose the proper base branch (*develop*)
- [ ] Confirm that the changes follow the project’s coding guidelines
- [ ] Verify that the changes generate no compiler or linter warnings
- [ ] Perform a self-review of the changes
- [ ] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Confirm that new and existing unit tests pass locally
- [ ] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [ ] Stick to the project’s gitflow workflow
- [ ] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by Tamdav — APPROVED** [review]
looks good!

File: fastlane/Fastfile [hunk 1]
```diff
@@ -41,4 +41,37 @@ platform :android do
             check_superseded_tracks:    'false',
         )
     end
-end
+	
+	desc "upload Paintroid metadata only"
```
Comment (wslany) [inline fastlane/Fastfile:45]:
```suggestion
	desc "upload Paintroid screenshots"
```
right?

File: fastlane/Fastfile [hunk 2]
```diff
@@ -41,4 +41,37 @@ platform :android do
...
+	  package_name:               'org.catrobat.paintroid.test',
+	  track:                      'alpha',
+	  track_promote_to:           'production',
+	  json_key_data:              ENV['googlePlayJsonKey'],
+	  metadata_path:              './fastlane/metadata/android',
+	  skip_upload_apk:            'true',
+	  skip_upload_metadata:       'true',
+	  skip_upload_images:         'true',
+	  skip_upload_screenshots:    'true',
+	  skip_upload_changelogs:     'true',
```
Comment (wslany) [inline fastlane/Fastfile:74]: Is this still needed?

File: fastlane/Fastfile [hunk 3]
```diff
@@ -41,4 +41,21 @@ platform :android do
             check_superseded_tracks:    'false',
         )
     end
-end
+	
+	desc "upload Paintroid screenshots"
+	lane :upload_Screenshots_Paintroid do
+		upload_to_play_store(
+		  package_name:               'org.catrobat.paintroid.test',
```
Comment (wslany) [inline fastlane/Fastfile:48]:
```suggestion
		  package_name:               'org.catrobat.paintroid',
```
This determines to which app on the play store they are uploaded. Here we want it for the main app.

File: fastlane/Screengrabfile [hunk 4]
```diff
@@ -0,0 +1,16 @@
+use_adb_root(false)
+
+app_package_name('org.catrobat.paintroid.test')
```
Comment (wslany) [inline fastlane/Screengrabfile:3]:
```suggestion
app_package_name('org.catrobat.paintroid')
```
I guess, though I'm not sure.

File: fastlane/Screengrabfile [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+use_tests_in_packages(['org.catrobat.paintroid.test.espresso.screenshots'])
+tests_package_name('org.catrobat.paintroid.test')
```
Comment (wslany) [inline fastlane/Screengrabfile:5]: I'm not sure if it has to be changed here as well.

**Review by wslany — CHANGES_REQUESTED** [review]
Please change package name to main app.

File: fastlane/Fastfile (hunk 3 above)
Comment (ThorstenBandel) [reply fastlane/Fastfile]: Changed

File: fastlane/Screengrabfile (hunk 4 above)
Comment (ThorstenBandel) [reply fastlane/Screengrabfile]: Yeah that needed to be changed. Is changed

**Review by wslany — APPROVED** [review]

**Comment by wslany** [comment]
Thanks!

**Pull request merged**